### ISSUE TITLE
Acknowledge contributors

### ISSUE THREAD

**Issue by ivan-aksamentov** [neherlab/covid19_scenarios#109]
We are thankful to all our contributors and we want to acknowledge their efforts in readme and in the app itself.

Something like https://github.com/all-contributors/all-contributors would be wonderful.

Ideally :
 - [x] have an automatic tool that extracts Github logins of all contributors, types of their contributions and their avatars
 - [x] markdown snippet is then inserted to the readme section (with a PR bot)
 - [x] markdown or mdx file is generated and imported into the app on every build

**Comment by ivan-aksamentov** [comment]
Continuing https://github.com/neherlab/covid19_scenarios/issues/16

I've setup a section in README.md, and a separate `.md` file to include into the app. Plus a PR bot and and added `all-contributors-cli`. But it seems that we now need to populate the array of contributors and their contribution types somehow

Do you know a way how to automatically gather `allcontributors` data from the existing contributions?

`yarn all-contributors check` only gives a list of nicknames, but does not deduce the types of contributions.

See also:
 - https://allcontributors.org

cc @rsharvill @ruisaraiva19

**Comment by rsharvill** [comment]
@ivan-aksamentov I'm looking into this!

**Comment by ruisaraiva19** [comment]
> @ivan-aksamentov I'm looking into this!

Sorry for a late comment here, I'm actually making a script to add all contributors to date.

**Comment by ivan-aksamentov** [comment]
@rsharvill @ruisaraiva19 

These commands
```
yarn all-contributors add <nickname> <contribution_type...>
yarn all-contributors generate
```
should give the desired effect. It's just I don't know how to get all the names and contributions for the `add` command. There's probably no tool that can deduce the types from issues and PRs, it would meed machine learning with natural language processing and image recognition or something :)

So basically, I was looking for a tool that can facilitate the manual work of going through all the issues and PRs and I found
https://github.com/mntnr/name-your-contributors

This command:
``
GITHUB_TOKEN="<your github token>" npx name-your-contributors -u neherlab -r covid19_scenarios
```
produces a huge JSON, Well, at least it is better than clicking things on GitHub.

Let me know what you have in mind!

**Comment by ruisaraiva19** [comment]
@ivan-aksamentov I was just starting using that package 👌.

**Comment by ruisaraiva19** [comment]
@ivan-aksamentov @rsharvill  PR ready for review.

**Comment by ivan-aksamentov** [comment]
@ruisaraiva19 Merged!
Any ideas how to perform maintenance? How to not forget anyone and not to go crazy at the same time.

**Comment by ruisaraiva19** [comment]
@ivan-aksamentov yes, I'm going to add a GitHub action workflow with `issueCreation`, `issueComment`, `prCreation`, `prComment`, `prReview` triggers to automatically open new PR with updated contributors list 🙂.

**Comment by ivan-aksamentov** [comment]
Resolved in #217 and #263
